Langs/0000.lang

```diff
@@ -44,6 +44,7 @@ Supported=
 Not Supported=
 Disabled / Not Detected=
 Drive(s) Meet Requirements=Drive(s) Meet Requirements
+0 Drives Meet Requirements=0 Drives Meet Requirements
```

Comment:
Text after equal sign is not needed, I am not sure why it is here (also line 46 and 48